activiti-spring-boot/spring-boot-starters/activiti-spring-boot-starter/src/test/java/org/activiti/keycloak/KeycloakEnabledBaseTestIT.java

```diff
@@ -55,11 +55,11 @@ protected AccessTokenResponse authenticateUser() throws IOException {
 
     protected HttpHeaders getHeaders(String token) {
         HttpHeaders headers = new HttpHeaders();
-        headers.add("Authorization", "Bearer " + new String(token));
+        headers.add("Authorization", "Bearer " + token);
```

Comment:
If it were always the same token then I could add it to the template but when we pass keycloak auth we then get a temporary token which we use for the life of that particular test. In theory if we had a really long-running test we'd have to call authenticateUser() again and get a new token (which would be simple to do as the methods are there in the base test).